### ISSUE TITLE
About NCE

### ISSUE THREAD

**Issue by MajorDavidZhang** [salesforce/PCL#3]
Hi，
your work is really excellent! 
However, I have a question when going through the code. Why the InfoNCE and ProtoNCE are replaced by the cross entropy, instead of the way shown in the paper?

**Comment by LiJunnan1992** [comment]
Hi, the InfoNCE and ProtoNCE loss have the same mathematical form as cross-entropy.
